src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -57,6 +57,8 @@ public static UserStore getTestInstance(PersistentStorageAgent persistentStorage
   /** The in-memory list of Users. */
   private List<User> users;
 
+  private List<String> admin_usernames = Arrays.asList("ayliana", "Marouane", "jeremy", "marissa", "raymond");
```

Comment:
nitpicky thing: prefer camelCase variable names when writing code in Java (versus snake_case).